javagen/src/main/java/com/azure/autorest/mapper/ClientMapper.java

```diff
@@ -287,9 +265,48 @@ public Client map(CodeModel codeModel) {
             builder.protocolExamples(protocolExamples);
         }
 
+        if (settings.isGenerateLiveTests() && codeModel.getTestModel() != null) {
+            builder.liveTests(parseLiveTests(codeModel.getTestModel()));
+        }
+
         return builder.build();
     }
 
+    private List<LiveTests> parseLiveTests(TestModel testModel) {
```

Comment:
This part is the vanilla? Do we need a map here?

If we need, it should be in a Mapper class (ClientMapper is already too long).

Reply:
I'll move it to a separate mapper.

Reply:
Added two live test mappers, `LiveTestsMapper` for vanilla and `FluentLiveTestsMapper` for fluent.